src/Components/Rocket/RocketItem.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from 'react-bootstrap/Button';
+
+const RocketItem = (props) => {
+  const { rocket } = props;
```

Comment:
## Optional: 

To help you write less code when destructuring the rocket, you can do this 👇️
```
const RocketItem = ({ rocket }) => {}

```
instead of this 👇️

```
const RocketItem = (props) => {
  const { rocket } = props;
}

```